fabfile.py

```diff
@@ -33,10 +31,22 @@
     subscribe,
     unsubscribe,
     update_basic_packages,
-    update_packages,
     upstream_install,
     vm_create,
     vm_destroy,
     vm_list,
     vm_list_base,
 )  # flake8: noqa
+from automation_tools.repository import (
+    create_custom_repos,
+    delete_custom_repos,
+    disable_beaker_repos,
+    disable_repos,
+    enable_repos,
+    enable_satellite_repos,
+    manage_custom_repos,
+)  # flake8: noqa
+from automation_tools.utils import (
+    distro_info,
+    update_packages
+)  # flake8: noqa
```

Comment:
`# flake8:noqa` causes flake8 to disable linting for the entire file, whereas `# noqa` causes flake8 to disable linting on a single line. I'd suggest dropping the `flake8: ` prefix. Beware that doing this may reveal linting issues that have been hidden.